06-building-a-program-start-to-finish/writing-tests/output/test/spec/inverted-index-spec.js

```diff
@@ -0,0 +1,89 @@
+const InvertedIndex = require('../../src/InvertedIndex.js');
+const samples = require('../samples');
+const FileAPI = require('file-api');
```

Comment:
I believe your "FileAPI" variable should be in camelCase. This should also apply on your InvertedIndex class